src/components/navigation-header/components/home-page-content/index.tsx

```diff
@@ -5,18 +5,13 @@ import { NavigationHeaderDropdownMenu } from '..'
 import sharedNavStyles from '../../navigation-header.module.css'
 import s from './home-page-content.module.css'
 
-/**
- * Defines the navigation items for the main home page of the app. If this
- * becomes authorable, it can be lifted into another area of the codebase.
- */
-const HOME_PAGE_NAV_ITEMS = [
-  { label: 'Documentation', pathSuffix: 'docs' },
-  { label: 'Tutorials', pathSuffix: 'tutorials' },
-  { label: 'Install', pathSuffix: 'downloads' },
-]
-
 const HomePageHeaderContent = () => {
```

Comment:
Small request for a `TODO` with the relevant ticket, but otherwise all good here! 🙌🏻 

```suggestion
/**
 * @TODO update content to also show products "coming soon"
 * @see https://app.asana.com/0/1202110981600689/1202300536466714/f
 */
const HomePageHeaderContent = () => {
```